version/version.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+import (
+	"fmt"
+
+	"github.com/rogpeppe/go-internal/semver"
+	"k8s.io/apimachinery/pkg/version"
+)
+
+type serverVersion interface {
```

Comment:
This interface should be exported otherwise callers of `CheckMinimumVersion` won't know what's expected of them.

Reply:
Good one, fixed.